site/tests/integration/login-flow.test.ts

```diff
@@ -0,0 +1,201 @@
+import execa from "execa";
+import { expect, test } from "playwright-test-coverage";
+
+import { readValueFromRecentDummyEmail } from "../shared/dummy-emails";
+import { openLoginModal } from "../shared/nav";
+
+const emailInputSelector = '[placeholder="claude\\@example\\.com"]';
+const loginButtonSelector = "button[type=submit]:has-text('Log In')";
+const verificationCodeInputSelector = '[placeholder="your-verification-code"]';
+const accountDropdownButtonSelector = '[data-testid="account-dropdown-button"]';
+
+test("login works for an existing user (via verification code)", async ({
+  page,
+  isMobile,
+}) => {
+  await execa("yarn", ["exe", "scripts/seed-db.ts"]);
+
+  await page.goto("/");
+  const loginModal = await openLoginModal({ page, isMobile });
+
+  const emailInput = loginModal.locator(emailInputSelector);
+  const loginButton = loginModal.locator(loginButtonSelector);
+
+  await emailInput.fill("alice@example.com");
+  await loginButton.click();
+
+  await expect(loginModal.locator("text=Check your inbox")).toBeVisible();
+
+  const verificationCodeInput = loginModal.locator(
+    verificationCodeInputSelector,
+  );
+
+  await verificationCodeInput.fill(
+    await readValueFromRecentDummyEmail("Email verification code: "),
+  );
+  await verificationCodeInput.press("Enter");
+
+  const accountDropdownButton = page.locator(accountDropdownButtonSelector);
+  await expect(accountDropdownButton).toBeVisible();
+
+  await accountDropdownButton.click();
+  await expect(accountDropdownButton).toHaveText("A");
```

Comment:
probably makes sense for this check to happen before the click...

Reply:
Fair point! Fixed in https://github.com/blockprotocol/blockprotocol/pull/455